ui/raidboss/data/05-shb/dungeon/matoyas_relict.txt

```diff
@@ -0,0 +1,59 @@
+### Matoya's Relict
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0 "--Reset--" sync /00:0839:.*is no longer sealed/ window 100000 jump 0
+
+### Mudman
+# -p 547F:1011
+# -ii 368 5481 5487 548E 548F 5490 5492
+1000.0 "--sync--" sync /00:0839:Clayclot Cauldron will be sealed off/ window 1000,0
+# Loop
+1011.0 "Hard Rock" sync /:Mudman:547F:/ duration 6 window 0,70
```

Comment:
For timeline loops, would you mind writing out a fake loop with a jump rather than a large window on a previous sync? In other words, something like

```
1011 "Hard Rock"
... etc
1066.7 "Stone Age"
1080.0 "Hard Rock" duration 6 jump 1011.0 window 20,20
1090.0 "Petrified Peat" #commented out sync
... etc, up to 30 seconds of commented out syncs to show the loop ahead
```

See: https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/05-shb/dungeon/malikahs_well.txt#L57-L69 for an example.

See also: https://github.com/quisquous/cactbot/blob/main/docs/TimelineGuide.md#building-loops

This makes the timeline look continuous to users.  Otherwise, the timeline will appear to not have any more entries as it approaches the end,, and then suddenly when it jumps back to the beginning abilities will appear again.

Reply:
Sure, sorry for inconsistent!